tests/backup_helper.go

```diff
@@ -4600,6 +4600,32 @@ func AddLabelsToMultipleNamespaces(labels map[string]string, namespaces []string
 	return nil
 }
 
+func VerifyLabelsFromMultpleNamespaces(labels map[string]string, namespaces []string) error {
+	log.Infof("Verifying labels %v in namespaces %v", labels, namespaces)
+	for _, namespace := range namespaces {
+		err := VerifyNamespaceLabel(namespace, labels)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func VerifyNamespaceLabel(namespace string, labels map[string]string) error {
+	labelsInTheNamespace, err := Inst().S.GetNamespaceLabel(namespace)
+	if err != nil {
+		return err
+	}
+	log.Infof("Labels in namespace %v are %v", namespace, labelsInTheNamespace)
+
+	for key, value := range labels {
+		if labelsInTheNamespace[key] != value {
+			return fmt.Errorf("label %s is not present in namespace %s", key, namespace)
```

Comment:
What will happen if the key itself is not found ?
I am not sure about  the behaviour, can you once please try that scenario also, as it should not panic if key itself is not found

Reply:
If there isn't a key present then it will return the default value accordingly. For ex: for map[string]string if key is not present default value "", will be returned.

Reply:
Better to add a check `value, ok :=  labelsInTheNamespace[key]`

Reply:
@dbinnal-px let's change it as per @mkoppal-px's suggestion and then we are good to merge it.